### ISSUE TITLE
Is there a way to add serious cases?

### ISSUE THREAD

**Issue by menachemshapiro** [zbraniecki/covid19-timeseries#7]
Thanks for making this. I've been looking for something like this for a while. 

Is there a way to track the serious cases as well. I guess that would be defined as either hospitalization or ICU cases. With that information we can compare how various countries are responding to the crisis.

**Comment by zbraniecki** [comment]
Yeah, it just needs data. If you can find data per-country for hospitalized, it'll be easy to hook in.

**Comment by menachemshapiro** [comment]
https://www.worldometers.info/coronavirus/#countries  -- worldometers.info has a per country breakdown of serious/critical cases, but I'm not sure where they're getting their data from.

**Comment by zbraniecki** [comment]
it does, but I don't know how to retrieve the data in source form - JSON, XML or other machine readable format.
